spec/models/photo_uploader_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "carrierwave/test/matchers"
+require "rails_helper"
+describe PhotoUploader do
+  include CarrierWave::Test::Matchers
+  before do
+    PhotoUploader.enable_processing = true
+    @card = Card.new(user_id: 1)
+    @uploader = PhotoUploader.new(@card, :remote_photo_url)
+    @uploader.store!(File.open("#{Rails.root}/spec/support/files/smile.jpg"))
+  end
+  after do
+    PhotoUploader.enable_processing = false
+    @uploader.remove!
+  end
+  context 'the small version' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.